tests/apps/test_sc_show.py

```
@@ -264,3 +264,121 @@ def test_sc_show_ext(flags, monkeypatch, make_manifests, asic_gcd):
+                                     '-screenshot', '-tool', 'klayout'])
+    with patch('siliconcompiler.Project.show') as show:
+        show.return_value = "test.png"
+        assert sc_show.main() == 0
+        show.assert_called_once_with(str(test_file), extension=None, screenshot=True,
+                                     tool='klayout')
+
+
+@pytest.mark.timeout(90)
+def test_sc_show_nonexistent_file_with_design(monkeypatch):
+    '''Test sc-show with non-existent file and design specified.'''
+    nonexistent_file = "/tmp/nonexistent_file_12345.gds"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent_file_12345.gds"

(S108)